website/pages/docs/builders/hyperv/iso.mdx

```diff
@@ -52,6 +52,58 @@ power off the VM.
 
 @include 'common/ISOConfig.mdx'
 
+Examples:
+go-getter can guess the checksum type based on `iso_checksum` len.
+
+<Tabs>
+<Tab heading="JSON">
```

Comment:
Nice one ! Do you think it is possible to make a partial out of this one ? I think the tabs don't work in partials; but then I'd rather not have tabs yet ( and uses words instead ) and make those tags when this issue is fixed. Because repeating this all over the place will make updating/maintaining of that chuck of test harder;

Reply:
You're right, tabs don't work on partials that's why I had to do this 😞